semaphore/integration_tests.py

```diff
@@ -80,26 +84,54 @@ def get_trip_plans(environment, from_place, to_place, trip_date, arrive_by):
     return requests.get(url, params).json()
 
 
-def compare_plans(plan1, plan2):
+# alerts come in random order, which sometimes causes plan comparison to fail
+def sort_alerts(json):
+    plan = json.get("plan")
+    for itinerary in plan.get("itineraries"):
+        for leg in itinerary.get("legs"):
+            alerts = leg.get("alerts")
+            if alerts and len(alerts) > 1:
+                leg["alerts"] = sorted(alerts, key=lambda a: a["alertDescriptionText"])
```

Comment:
- you don't really need to check the length: sorting a single-element list is a no-op
- should it sort by ID instead?

Reply:
- ok, removed the check;
- it would be better, but there is no ID field in the response

```json
            "alerts": [
              {
                "alertDescriptionText": "This change is part of the first round of Better Bus Project service updates going into effect this year. \r\n\r\nChanges like this will help improve the reliability and flexibility of our bus system, and are based on feedback from over 3,000 MBTA bus riders like you, along with planners, drivers and consultants.\r\n\r\nYou can learn more about how were improving bus service throughout the region at mbta.com/betterbus",
                "effectiveStartDate": 1564571141000
              },
              {
                "alertDescriptionText": "Affected routes:\r\n94\r\n95\r\n96\r\n101\r\n134\r\n326",
                "effectiveStartDate": 1559550600000
              }
            ],
```